core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -769,6 +796,43 @@ private[spark] class MapOutputTrackerWorker(conf: SparkConf) extends MapOutputTr
   }
 }
 
+/**
+ * MapOutputTrackerWorker for continuous processing, the main difference with MapOutputTracker
+ * is waiting for a time when the upstream's map output status not ready.
```

Comment:
We can't change the behavior for queries which aren't continuous processing.

Reply:
Maybe we didn't change here, IIUC in batch mode, child stage only submitted while no missing partition of parent stage.

Reply:
We can't really make changes that maybe don't change existing behavior. They need to definitely not change existing behavior, ideally by not using the same codepath if they need to do different things.

Reply:
The current implement of ContinuousShuffleMapOutputTracker didn't change existing behavior, just choose the corresponding output tracker instance by the job type. After we use a better way of mark a job is continuous mode(discuss in https://github.com/apache/spark/pull/21293#discussion-diff-187368178R231), they do different things and not use same codepath.